reflection_refraction.py

```diff
@@ -41,8 +43,11 @@ def calculate_refl_angl(alfa, n1, n2):
     if tot_intern_refl(alfa, n1, n2):
         beta = 90
     else:
-        beta = math.degrees(math.asin(n1*math.sin(math.radians(alfa))/n2))  # n1*sin(θ1) = n2*sin(θ2)
-    
+        try:
+            beta = math.degrees(math.asin(n1*math.sin(math.radians(alfa))/n2))  # n1*sin(θ1) = n2*sin(θ2)
+        except ValueError:
+            beta = None
```

Comment:
Whet I add exception _beta = None_ it can't sum beta after that and it's giving me error. Can I remove beta from the things that the program calculates without rewriting it, because I think it's too much work???

Reply:
The way you handle `if beta is not None` below is fine.